models/tasktemplate/fromdir.go

```diff
@@ -50,5 +60,6 @@ func LoadFromDir(dbp zesty.DBProvider, dir string) error {
 		}
 		logrus.Infof("%s task template '%s'", verb, tt.Name)
 	}
+	templateimport.CleanTemplates()
```

Comment:
Why not leave the list intact?
If you clean it, you cannot call subtask.ValidConfig ever during the runtime, which feels a bit weird/brittle

Reply:
subtask.ValidConfig at runtime should validate only against template in database, because that's the template we can use. The templateimport collection is just here to registers template names for startup.

I chose to clean up after the loop here to avoid any side effect inside the software of code path that will still use template names for the boot collection instead from the database.

As the Valid func in subtask plugin is looking for template inside database AND the templateimport collection, runtime validation will be okay.